src/controllers/token.rs

```diff
@@ -199,3 +218,25 @@ pub async fn revoke_current(app: AppState, req: Parts) -> AppResult<Response> {
     })
     .await
 }
+
+struct NewTokenEmail<'a> {
+    user_name: &'a str,
+    domain: &'a str,
+}
+
+impl<'a> crate::email::Email for NewTokenEmail<'a> {
+    const SUBJECT: &'static str = "New API token created";
+
+    fn body(&self) -> String {
+        format!(
+            "\
+Hello {user_name}!
+
+A new API token was recently added to your crates.io account.
```

Comment:
Should this `crates.io` reference use `{domain}` too?

Reply:
I guess that would make sense, yeah